edgelet/support-bundle/Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [package]
-name = "support-bundle"
-version = "0.1.0"
 authors = ["Lee Fitchett <lefitche@microsoft.com>"]
 edition = "2018"
+name = "support-bundle"
+version = "0.1.0"
 
 [dependencies]
 chrono = "0.4.7"
 failure = "0.1"
-futures = "0.1"
-tokio = "0.1"
-zip = "0.5.3"
+futures = "0.3"
+tokio = {version = "1", features = ["process"]}
```

Comment:
spacing nit: should be `{ version = "1", features = ["process"] }`

Same for lines 14 and 17 below.